src/biotite/application/foldx/app.py

```diff
@@ -0,0 +1,190 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.application.foldX"
+__author__ = "Mojmir Mutny"
+__all__ = ["FoldXApp"]
+
+import copy
+from tempfile import NamedTemporaryFile
+import numpy as np
+import os 
+from os import chdir, getcwd, remove
+from ..localapp import LocalApp, cleanup_tempfile
+from ..application import AppState, requires_state
+from ...structure.io.pdbqt import PDBQTFile
+from ...structure.io.pdb import PDBFile
+from ...structure.io.pdbx import PDBxFile
+from ...structure.residues import get_residue_starts_for, get_residue_masks
+from ...structure.bonds import find_connected
+from ...structure.error import BadStructureError
+
+
+class FoldXApp(LocalApp):
```

Comment:
As I read so far, FoldX has multiple subcommands, and the one used here is called `BuildModel`. Hence, i propose the following:
```suggestion
class BuildModelApp(LocalApp):
```
This is consistent with e.g. the `viennarna` subpackage, where the classes are called `RNAfoldApp` and `RNAplotApp`.